roles/lib_openshift/src/test/unit/conftest.py

```diff
@@ -0,0 +1,55 @@
+import inspect
+
+import pytest
+
+from lib_openshift.library import *  # noqa: F401,F403
+
+
+def pytest_generate_tests(metafunc):
+    if 'binary_lookup_module' in metafunc.fixturenames:
+        lookup_modules = []
+        ids = []
+        for key in list(globals().keys()):
+            value = globals()[key]
+            if inspect.ismodule(value):
+                for func, _ in inspect.getmembers(value, inspect.isfunction):
+                    if func == 'locate_oc_binary':
+                        lookup_modules.append(value)
+                        ids.append(key)
+        metafunc.parametrize('binary_lookup_module', lookup_modules, ids=ids)
+
+
+@pytest.fixture
+def mock_tmpfile_copy(request, mocker):
+    module_under_test = request.module.MODULE_UNDER_TEST
+    patch_method = module_under_test.__name__ + '.Utils.create_tmpfile_copy'
+    mocker.patch(patch_method, return_value='/tmp/mocked_kubeconfig')
+    yield
```

Comment:
I don't think it's needed anymore, I was using the context manager from mocker originally and left the yield after it was refactored.